dexbot/strategies/base.py

```diff
@@ -635,9 +655,27 @@ def get_market_buy_price(self, quote_amount=0, base_amount=0):
 
         return base_amount / quote_amount
 
-    def get_market_orders(self, depth=1):
+    def get_limit_orders(self, depth=1, updated=True):
```

Comment:
I wouldn't change this name. All market orders in bitshares are limit orders, aren't they? If they aren't, then we are also interested in the ones that aren't. The name "market order" implies that we are interested in any order that can be found on the market, and the method name is thus descriptive and understandable. "limit" bears no meaning separate from "market" in BitShares, and so I would use the one that in my opinion is more intuitive and descriptive: market.